skorch/callbacks/logging.py

```diff
@@ -231,18 +231,27 @@ def on_epoch_end(self, net, **kwargs):
 
     def on_train_end(self, net, **kwargs):
         try:
-            self._metric_logger['train/epoch/event_lr'].log(net.history[:, 'event_lr'])
+            self._metric_logger['train/epoch/event_lr'].append(net.history[:, 'event_lr'])
         except KeyError:
             pass
         if self.close_after_train:
-            self.run.stop()
+            try:  # >1.0 package structure
+                from neptune.handler import Handler
+            except ImportError:  # <1.0 package structure
+                from neptune.new.handler import Handler
```

Comment:
This can be just

```
from neptune.handler import Handler
```

given that the dependency is v >=1.0.0

Reply:
This is actual logging code which should be compatible with both pre and post 1.0. 

Other update where we assume 1.0 structure is in test (where we have updated the requirements to install latest version).

Reply:
But the requirements say `neptune` hence >=1.0.0, so there is no backwards compatibility assumed.